techs/bem-xjst.js

```diff
@@ -128,7 +128,14 @@ module.exports = buildFlow.create()
                 engineOptions.naming = this._naming;
             }
 
-            engineOptions.exportName = engineOptions.exportName || this._exportName;
+            engineOptions.exportName || (engineOptions.exportName = this._exportName);
+
+            if (this._requires) {
+                this.node.getLogger().logOptionIsDeprecated(this.node.unmaskTargetName(this._target),
+                    'enb-bemxjst', this.getName(), 'requires', 'engineOptions.requires',
+                    ' It will be removed in v9.0.0.');
```

Comment:
How not to forget about it when releasing a v9.0.0?

Reply:
Created issue https://github.com/enb/enb-bemxjst/issues/287